docs/Testing.md

```diff
@@ -151,15 +151,20 @@ We first need to compile the tool, which can be done in one of two modes.
 In the first mode, the tool is compiled with a statically linked model.
 This mode requires the `-D LOAD_MODEL_STATICALLY=0` option during compilation in addition to including the `.so` file.
 Best is to use the `build-run-onnx-lib.sh` script in the `onnx-mlir/utils` directory to compile the tool with its model, which is passed as a parameter to the script.
-To avoid library path issues, just run the tool in the home directory of the model.
+To avoid library path issues on Mac, run the compiled tool in the directory where the model was built.
 
 ``` sh
 # Compile tool with model.
 cd onnx-mlir/build
-. ../utils/build-run-onnx-lib.sh test/backend/test_add.so
-# Run tool in the directory of the model.
-(cd test/backend; run-onnx-lib)
+sh ../utils/build-run-onnx-lib.sh test/backend/test_add/test_add.so
+# Run the tool to run the model.
+Debug/bin/run-onnx-lib
+# or, on Mac, run the tool in the directory where the model was built
+(cd test/backend; ../../Debug/bin/run-onnx-lib)
+# if test_add.so was built in `test/backend`:
+cd test/backend; ../../Debug/bin/onnx-mlir --EmitLib test_add/test_add.onnx
 ```
+(You can see the path of the library with `otool -L test_add.so` on Mac.)
```

Comment:
I tested on Mac and Linux

on Linux things worked fine without changing directory, so I'm guessing that this is a Mac specific problem and I edited the explanation accordingly, and added the Mac-specific `otool` advice

please let me know if anyone is aware of this problem existing on other platforms than Mac